src/main/java/nl/lucasridder/lhub/commands/Gamemode.java

```diff
@@ -22,25 +23,25 @@ public boolean onCommand(CommandSender sender, Command cmd, String label, String
 
             //te weinig args
             if(args.length == 0 | args.length > 2) {
-                player.sendMessage(ChatColor.RED + "/gamemode (creative/survival/spectator/adventure/0/1/2/3) (speler)");
+                player.sendMessage(ConfigManager.getString(Util.color("messages.gamemodeUsage")));
```

Comment:
You probably need to swap around the `ConfigManager.getString` and `Util.color` calls here (and in all the following calls), as currently you are fixing the color codes on the `"messages.gamemodeSucces"` string instead of on the config value returned by `getString`.

```suggestion
                player.sendMessage(Util.color(ConfigManager.getString("messages.gamemodeUsage")));
```